libs/sdk-ui-ext/src/internal/utils/uiConfigHelpers/baseChartUiConfigHelper.ts

```diff
@@ -156,3 +165,28 @@ export function setBaseChartUiConfigRecommendations(
     }
     return referencePoint;
 }
+
+function getBucketItemsIcons(bucket: IBucketItem[]): string {
+    const attributeUsed = bucket.find((x) => !isDateBucketItem(x));
+    const dateUsed = bucket.find((x) => isDateBucketItem(x));
+
+    if (attributeUsed && dateUsed) {
+        return '<span class="attr-field-icon" /> / <span class="date-field-icon" />';
```

Comment:
Is UX ok with / instead of translated "or" ?

Reply:
Approved

Reply:
Changed to OR